tester.py

```diff
@@ -92,11 +94,19 @@ def update_validation_test(random = True):
     Only assumes that update exists - no web/DB assumptions
     made.
     """
+    token = str(uuid.uuid4())
+
+    update_val = {
+    }
     from validate import validate_updates
     #a different fake user (I waste so much time).
     fake_usr = {
         "_id": "The Mongo Longo",
         "email": "doesnt@matter.horn",
+        "auth": {
+            "token": token,
+            "valid_until": (datetime.now() + timedelta(days=3)).isoformat()
+        },
```

Comment:
Note: this won't do anything. (I don't care because I'll deprecate this file ASAP.)